ign-rviz-plugins/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>ign-rviz-plugins</name>
+  <version>0.0.0</version>
+  <description>This package contains visualization plugins for ign-rviz</description>
+
+  <maintainer email="sarathkrishnan99@gmail.com">Sarathkrishnan Ramesh</maintainer>
+
+  <license>TODO</license>
+
+  <build_depend>rclcpp</build_depend>
```

Comment:
something which is <build_depend>, <build_export_depend> and <exec_depend> you just use <depend> https://www.ros.org/reps/rep-0149.html#depend-multiple